opslevel/common.go

```diff
@@ -342,8 +344,11 @@ func NewFilterBlockModel(field string, value string) filterBlockModel {
 func FilterAttrs(validFieldNames []string) map[string]schema.Attribute {
 	filterAttrs := map[string]schema.Attribute{
 		"field": schema.StringAttribute{
-			Description: "The field of the target resource to filter upon.",
-			Required:    true,
+			Description: fmt.Sprintf(
+				"The field of the target resource to filter upon. One of `%s`",
+				strings.Join(validFieldNames, "`, `"),
```

Comment:
I think this is a mistake

```suggestion
				strings.Join(validFieldNames, ", "),
```

Reply:
No this was intentional. Wrapping each of these options in backticks gives the docs a nice markdown feature. Example of this used in [code here](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/resource_opslevel_trigger_definition.go#L80) and shown in [docs here](https://registry.terraform.io/providers/OpsLevel/opslevel/latest/docs/resources/trigger_definition#access_control)

Reply:
I thought this would double quote with backticks but it's correct